File: BusanSubway/app/src/main/java/dsu/software/busansubway/Inside/FirebaseHandler.java
package dsu.software.busansubway.Inside;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;

import java.util.ArrayList;

import dsu.software.busansubway.R;

public class FirebaseHandler extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inside_detail_seat);

        // Write a message to the database
        FirebaseDatabase database = FirebaseDatabase.getInstance();





    }
}

File: BusanSubway/app/src/main/java/dsu/software/busansubway/Inside/FirebaseHandler.kt
package dsu.software.busansubway.Inside

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.database.FirebaseDatabase
import dsu.software.busansubway.R

class FirebaseHandler : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_inside_detail_seat)

        // Write a message to the database
        val database = FirebaseDatabase.getInstance()
    }
}